session-1/session1.html

```diff
@@ -0,0 +1,113 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- CSS Bootsrtap -->
+    <link rel="stylesheet" href="assets/css/bootstrap.min.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <nav class="navbar navbar-expand-lg fixed-top navbar-light bg-light">
+            <div class="container">
+                <a class="navbar-brand" href="#">
+                    <img src="assets/img/Wikimedia-logo.png" alt="" width="30" class="d-inline-block align-text-top"> Danjou
+                </a>
+                <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+                <div class="collapse navbar-collapse" id="navbarNav">
+                    <ul class="navbar-nav ms-auto">
+                        <li class="nav-item">
+                            <a class="nav-link active" aria-current="page" href="#">About</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" href="#">Experience</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" href="#">Skills</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link active" href="#">Portofolio</a>
+                        </li>
+                        <li class="nav-item">
+                            <a href="#" class="nav-link active btn btn-success rounded" style="color: #fff;">Contact</a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <!-- Content -->
+    <section class="p-5 py-5 mt-5">
+        <div class="container">
+            <div class="row">
+                <div class="col-12 mb-3">
+                    <div class="row flex-row-reverse">
+                        <div class="col-md-6 col-sm-6 card border-0 mb-3 shadow-sm">
+                            <img src="assets/img/3.png" class="img-fluid w-100" alt="">
+                        </div>
+                        <div class="col-md-6 col-sm-6 mb-3 align-self-center">
+                            <h1 class="fw-bold">Hy, I'm Lorem Ipsum</h1>
+                            <p>This Website tell me about my portfolio and CV, to make it easier for you to assess my skills and personality</p>
```

Comment:
Add **display-(value)** class to make the font bigger. Here is the guidance [**Display Heading**](https://getbootstrap.com/docs/5.1/content/typography/#display-headings)